src/core/trulens/core/app/base.py

```diff
@@ -712,8 +712,8 @@ def _tru_post_init(self):
                     "Feedback logging requires `tru` to be specified."
                 )
 
-        if self.feedback_mode == mod_feedback_schema.FeedbackMode.DEFERRED:
-            for f in self.feedbacks:
+        for f in self.feedbacks:
```

Comment:
why this change? `self.feedback_mode` should be constant

Reply:
oh thanks, I forgot to add the other constraint (if it's a SnowflakeFeedback).